gentoo-quick-installer.sh

```diff
@@ -11,23 +11,41 @@
 # export OPTION2=VALUE2
 # ./gentoo-quick-installer.sh
 #
+# livecd kernel with root password
+# USE_LIVECD_KERNEL=1 ROOT_PASSWORD=Gentoo123 ./gentoo-quick-installer.sh
+#
+# Gentoo's binary kernel with ssh RSA public key
+# USE_LIVECD_KERNEL=0 SSH_PUBLIC_KEY=$(cat id_rsa.pub) ./gentoo-quick-installer.sh
 # Options:
 #
 # USE_LIVECD_KERNEL - 1 to use livecd kernel (saves time) or 0 to build kernel (takes time)
 # SSH_PUBLIC_KEY - ssh public key, pass contents of `cat ~/.ssh/id_rsa.pub` for example
 # ROOT_PASSWORD - root password, only SSH key-based authentication will work if not set
+#
+# Notes:
+# This script will _only_ work on a mbr/msdos partition table, not GPT. 
+# This script does not work with UEFI. only Legacy BIOS.
 ##
 
 set -e
 
 GENTOO_MIRROR="http://distfiles.gentoo.org"
 
 GENTOO_ARCH="amd64"
-GENTOO_STAGE3="amd64"
-
-TARGET_DISK=/dev/sda
-
-TARGET_BOOT_SIZE=100M
+# Default is amd64 build with OpenRC.
+# Other options are amd64-desktop-openrc, amd64-nomultilib-openrc,
+# amd64-hardened-selinux-openrc, amd64-musl, md64-musl-hardened, 
+# amd64-hardened-selinux-openrc, amd64-hardened-openrc, 
+# amd64-hardened-nomultilib-selinux-openrc, and amd64-hardened-nomultilib-openrc
+GENTOO_STAGE3="amd64-openrc"
+
+# Default is set for virtual machines.
+# /dev/sda is standard for most IDE/SATA drives.
+# /dev/nvme0n1 is standard for NVME drives.
+# /dev/mmcblk0 is standard for most eMMC and SD drives
+TARGET_DISK=/dev/vda
```

Comment:
Might make it `TARGET_DISK=${TARGET_DISK:-/dev/sda`.
This script was create for installation in VirtualBox. We can keep default for backward compatibility but still allowing to override disk name.

Reply:
The issue with this idea is the weird hybrid it creates with the script, where you're either expected to hand-edit it (GENTOO_STAGE3 and TARGET_DISK) to make sure it works, and issue envars while running the command afterwards. for the purposes of automation and being run from other scripts, hand-editing scripts is less desireable. 

I'll change both TARGET_DISK and GENTOO_STAGE3 for now.

Reply:
Agree